editor/src/core/shared/code-exec-utils.ts

```diff
@@ -223,7 +223,7 @@ export const SafeFunctionCurriedErrorHandler = {
   },
 }[UTOPIA_FUNCTION_ROOT_NAME]
 
-export function SafeFunction(
+export function safeFunction(
```

Comment:
why the rename?

Reply:
https://github.com/vitejs/vite/issues/6148